modules/mytargetBidAdapter.js

```diff
@@ -0,0 +1,112 @@
+import * as utils from '../src/utils.js';
```

Comment:
Can you please only import the functions needed from utils here? We just went through all adapters this week to eliminate the need for `utils.`

Reply:
@ChrisHuie i fixed this import but tests failed again, local test OK!
In First I merge topical master